src/userstyles.yml

```diff
@@ -414,4 +417,13 @@ userstyles:
           answer: "It is there in-case you have an OLED display. If you have one, you might want to enable this."
       current-maintainers: [*isabelroses]
       past-maintainers: [*elkrien]
+  syncthing:
+    name: Syncthing
+    category: productivity
+    readme:
+      app-link: "https://syncthing.net"
+      usage: |+
+        > **Note** <br>
+        > This theme is designed to be used with the default WebGUI and default theme provided by Syncthing. This doesn't theme syncthing.net
+      current-maintainers: [*BlankParticle]
```

Comment:
Can you move this so its listed alphabetically like the other ports e.g. anilist, boring-proxy, etc

Reply:
done